app/models/ingredient.rb

```diff
@@ -3,7 +3,18 @@
 class Ingredient < ApplicationRecord
   validates :name, uniqueness: true
 
+  MEASURES = %w[cup oz ml fl oz
+                g kg piece leaf
+                pinch lb tsp tbsp].map { |measure| [measure.capitalize, measure] }.sort
+
   CATEGORIES = %w[dairy fruits spices meats
                   flour alcohol cereals liquids
                   nuts vegetables other].map { |category| [category.capitalize, category] }.sort
+
+  def grouped_ingredients
+    CATEGORIES.map do |item|
+      ingredients_in_category = Ingredient.where(category: item).order(name: :desc).pluck(:name)
+      [item[0], [ingredients_in_category.map(&:capitalize), ingredients_in_category]]
+    end.collect
```

Comment:
What are you trying to do here? It seems kinda query intensive. 💀

Reply:
grouped_ingredients yields the contents for a select I will be using in a feature called inventory. That select is the one I asked about [here](https://michelada.slack.com/archives/CEAAXTYR5/p1643070195018900) :3

Reply:
I thought categories was meant to be a resource managed by an admin 👀

Reply:
Also I don't see where you using this methods. I got the feeling you're including things don't need right now, if that's the case let's get rid of this method. You can include it at the moment you need it

Reply:
I-m removing them in my next commit

Reply:
This is being removed in the latest commit.